packages/core/src/lib/markdown-it/patches/custom-component/htmlBlockRule.js

```diff
@@ -0,0 +1,147 @@
+const blockNames = require('markdown-it/lib/common/html_blocks');
+const { HTML_OPEN_CLOSE_TAG_RE } = require('./htmlRe');
+const inlineTags = require('./inlineTags');
+const vueReservedTags = require('./vueReservedTags');
+
+function initCustomComponentHtmlBlockRule(tagsToIgnore) {
+  // MODIFIED (MarkBind): Added 'special tags' and 'minimal panel' related rules
+
+  /* 
+   * Set up special tags that we need to ignore for markdown parsing.
+   * Custom patch for the api to escape content in certain special tags.
+   * 
+   * script, pre, style are for Vue
+   */
+  const specialTagsRegex = Array.from(tagsToIgnore)
+  .concat(['include|variable|site-nav|markdown|md|script|pre|style'])
+  .join('|');
+
+  // attr_name, unquoted ... attribute patterns adapted from markdown-it/lib/common/html_re
+  // Construct self-closing negative lookahead for special tags,
+  // because if matched, the endingSpecialTagRegex will roll down to find a non-existent closing tag
+  const attr_name = '[a-zA-Z_:][a-zA-Z0-9:._-]*';
+  const unquoted      = '[^"\'=<>`\\x00-\\x20]+';
+  const single_quoted = "'[^']*'";
+  const double_quoted = '"[^"]*"';
+  const attr_value  = '(?:' + unquoted + '|' + single_quoted + '|' + double_quoted + ')';
+  const attribute   = '(?:\\s+' + attr_name + '(?:\\s*=\\s*' + attr_value + ')?)';
+  const selfClosingNegativeLookahead = `(?!${attribute}*\\s*\/>)`;
+
+  const startingSpecialTagRegex = new RegExp(
+  `^<(${specialTagsRegex})${selfClosingNegativeLookahead}(?=(\\s|>|$))`, 'i');
+  const endingSpecialTagRegex = new RegExp(`<\\/(${specialTagsRegex})>`, 'i');
+
+  /*
+  * In MarkBind, we consider minimized panel components to be inline elements.
+  * Thus, we have to ensure that markdown-it parses minimized panel as inline element
+  * by using the regex below. 
+  */
+  const minimizedPanelStringRegex = '(panel.*minimized.*)';
```

Comment:
```suggestion
  const minimizedPanelStringRegex = '(panel${attribute}*minimized)';
```
would this be better instead?
- `${attribute}*` preceding `minimized` to ensure its not part of an attribute value
- removal of second `.*` handled since it should be handled by vuepress' regex

let's add a few positive / negative tests for this as well.

Reply:
Thanks for the suggestion :-) I went with the former `*minimized.*` because I figured the chances of this edge case is rather slim. However, the regex that you suggested works really well and seems to cover the edge case sufficiently, if not fully. I've added a few test cases using the example that you provided in #958. Let me know if I missed out other edge cases!